### PR TITLE
Version 0.18.0

### PR DESCRIPTION
**Pull request by andyone**
#### New Features
* `[cli]` Added ability to delete some ruby versions (#64)
* `[cli]` Added error messages about used conflicts options

#### Improvements
* `[cli]` Minor UI improvements
* `[cli|gen|clone]` ek package updated to latest version
* Improved Makefile

### CODE REVIEW

File: cli/cli.go [hunk 1]
```diff
@@ -739,6 +795,36 @@ func rehashShims() {
 	}
 }
 
+// checkHashTaskHandler check archive checksum
```
Comment (gongled) [inline cli/cli.go]: It looks like a typo. Use `unistallTaskHandler` instead.

File: cli/cli.go [hunk 2]
```diff
@@ -716,13 +721,64 @@ func installCommand(rubyVersion string) {
...
-		fmtc.Printf("{g}Version {g*}%s{g} successfully installed as {g*}%s{g}.{!}\n", info.Name, cleanVersionName)
+		fmtc.Printf("{g}Version {*}%s{!*} successfully installed as {*}%s{!}\n", info.Name, cleanVersionName)
 	} else {
 		log.Info("[%s] Installed version %s", currentUser.RealName, info.Name)
-		fmtc.Printf("{g}Version {g*}%s{g} successfully installed.{!}\n", info.Name)
+		fmtc.Printf("{g}Version {*}%s{!*} successfully installed{!}\n", info.Name)
 	}
 }
 
+// unistallCommand unistall some version of ruby
```
Comment (gongled) [inline cli/cli.go]: Please, fix the typo in word install (here is `unistall`).

**Comment by andyone** [comment]
Oh, thank you for noticing this!

**Comment by andyone** [comment]
Fixed in 909a95d051ca0a1c0eafc7d6bb1247af206d3870

**Pull request merged**